nextflow_schema.json

```diff
@@ -4,7 +4,7 @@
     "title": "nf-core/test pipeline parameters",
     "description": "testing",
     "type": "object",
-    "defs": {
+    "$defs": {
```

Comment:
if `$defs` is correct, something's wrong with the migration guide cc @ewels

Reply:
yup: the replacement in the automatic migration script uses `$defs`, not `defs`

![Screenshot 2024-07-10 at 09 18 37](https://github.com/seqeralabs/nf-schema-testing/assets/18287793/9454cb01-bb69-4aaa-b476-2f1f25a71e04)